src/components/PatchActionButtons/RestartPatch.tsx

```diff
@@ -36,7 +36,7 @@ export const RestartPatch = forwardRef<HTMLDivElement, RestartPatchProps>(
           </DropdownItem>
         )}
         {openModal && (
-          <div className={openModal && "ant-popover-open"} ref={ref}>
+          <div className={openModal ? "ant-popover-open" : ""} ref={ref}>
```

Comment:
I know you didn't add this class, but do you know what it does? I couldn't find anything about it from a search in the antd docs. And it doesn't look like a style is being applied to the element. Could we remove this?

Reply:
the className is added when `openModal` is true so the parent component can see when the modal is open by checking `ref.current.className` instead of listening to changes on `openModal` variable. https://github.com/evergreen-ci/spruce/blob/master/src/pages/userPatches/patchCard/DropdownMenu.tsx#L33
I tried removing the className and the modal closed on click so I think we need it for now.